esp-hal/src/dma/gdma.rs

```diff
@@ -94,24 +94,41 @@ static RX_WAKERS: [AtomicWaker; CHANNEL_COUNT] = [const { AtomicWaker::new() };
 impl<C: GdmaChannel> crate::private::Sealed for ChannelTxImpl<C> {}
 
 impl<C: GdmaChannel> ChannelTxImpl<C> {
+    #[cfg(not(esp32p4))]
```

Comment:
Can we just cfg away DMA changes (i.e remove gdma from metadata)? We won't have a need for it for some time, and DMA on P4 is going to be complicated.

Reply:
if that works: https://github.com/esp-rs/esp-hal/pull/2466#discussion_r1831138256

`gdma` will be temporarily removed from metadata